models/tasks.js

```diff
@@ -49,6 +49,28 @@ const fetchTasks = async () => {
   }
 }
 
+/**
+ * Fetch all participants whose task status is active
+ *
+ * @return {Promise<tasks|Array>}
+ */
+
+const fetchActiveTaskMembers = async () => {
```

Comment:
Unit tests pending for this function.
Can you add its tests in `test/unit/models/tasks.test.js`?

Reply:
I will create new issue for these test cases